spec/index.html

```diff
@@ -355,59 +355,61 @@ <h3>Blank nodes</h3>
-    a date of origin:</p>
-    <pre class="example nohighlight">
-      &lt;Bob's interest> &lt;is an> &lt;Interest> .
-      &lt;Bob's interest> &lt;since> &lt;4th of October 1998> .
-    </pre>
+    <figure id="fig3">
+      <a href="example-triple-annotation.svg" data-include="example-triple-annotation.svg">
+      </a>
+      <figcaption>Informal triple annotation example: a concretization of a
+      statement described with its own characteristics.</figcaption>
+    </figure>
 
-    <p>Formally, an annotation is formed by two triples: the statement itself
-    and a <a data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying
-    triple</a>. This expresses the simple relationship and the reifying relation to that from the circumstance, as a concretization thereof:</p>
+    <p>In conjunction to the arc expressing the relationship, the dashed circle
+    in the figure expresses a <a
+    data-cite="RDF12-CONCEPTS#dfn-reifier">reifier</a>. This is Bob's interest,
+    with a type and a date of origin, as a concrete circumstance of the fact
+    that Bob is interested in the Mona Lisa.</p>
+
+    <p>Formally, this annotation is composed of four triples:</p>
     <pre class="example nohighlight">
       &lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt; .
       &lt;Bob's interest> &lt;is a concretization of> <strong>&laquo;&lpar; <small>&lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt;</small> &rpar;&raquo;</strong> .
+      &lt;Bob's interest> &lt;is an> &lt;Interest> .
+      &lt;Bob's interest> &lt;since> &lt;4th of October 1998> .
     </pre>
 
-    <p><a data-cite="RDF12-CONCEPTS#dfn-reifier">Reifiers</a> are subjects of
-    triples relating to triple terms using the <code>rdf:reifies</code>
-    predicate. Triple terms should only appear in the <strong>object
-    position</strong> of such reifying triples.</p>
-
-    <p>Note that triple terms only <em>reference</em> statements. This means
-    that we can describe statements without implying them. This is useful for
-    describing unknown propositions, such as suggestions or disputed claims. We
-    can also annotate statements with multiple reifiers, to describe different
-    circumstances varying in time, location, etc.</p>
-
-    <aside class="note">Triple terms were added to the RDF data model in RDF 1.2.</aside>
+    <p>The second triple is called a <a
+    data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying triple</a>, and
+    expresses a concretization of the proposition stated by the first triple.
+    The object of a reifying triple is a <a
+    data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>. It denotes the
+    proposition itself, as a logical, abstract object identified through its
+    constituent subject, predicate and object components.</p>
+
+    <p>Triple terms may only appear in the <strong>object position</strong>,
+    and should be used with the special <em>reifies</em> predicate of reifying
```

Comment:
Is this `should` normative?
```suggestion
    and SHOULD be used with the special <em>reifies</em> predicate of reifying
```

Reply:
The primer isn't a REC, it is a W3C Working Group Note (at least RDF 1.1 was). The text isn't normative and it does not have the activating text: "The key words ... in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119)."

"should" is fine.

The point of RFC2119 is to have emphasis when a word has spec-significance.

Reply:
I agree to keep "should" (it is also used in other places in the text). @TallTed do you accept that? (Feel free to just "Resolve conversation" if so. :slightly_smiling_face: )

Reply:
I'll live with it. I guess no harm will come if implementers act like `should` is normative.